scenes/SceneEditAccountV2.js

```diff
@@ -0,0 +1,411 @@
+import * as React from "react";
+import * as System from "~/components/system";
+import * as Actions from "~/common/actions";
+import * as Strings from "~/common/strings";
+import * as Validations from "~/common/validations";
+import * as FileUtilities from "~/common/file-utilities";
+import { css } from "@emotion/react";
+import { dispatchCustomEvent } from "~/common/custom-events";
+import { TabGroup } from "~/components/core/TabGroup";
+
+import ScenePage from "~/components/core/ScenePage";
+import ScenePageHeader from "~/components/core/ScenePageHeader";
+import Avatar from "~/components/core/Avatar";
+
+const STYLES_FILE_HIDDEN = css`
+  height: 1px;
+  width: 1px;
+  opacity: 0;
+  visibility: hidden;
+  position: fixed;
+  top: -1px;
+  left: -1px;
+`;
+
+const STYLES_COPY_INPUT = css`
+  pointer-events: none;
+  position: absolute;
+  opacity: 0;
+`;
+
+export default class SceneEditAccount extends React.Component {
+  _ref;
+
+  state = {
+    username: this.props.viewer.username,
+    password: "",
+    confirm: "",
+    body: this.props.viewer.data.body,
+    photo: this.props.viewer.data.photo,
+    name: this.props.viewer.data.name,
+    deleting: false,
+    allow_filecoin_directory_listing: this.props.viewer.allow_filecoin_directory_listing,
+    allow_automatic_data_storage: this.props.viewer.allow_automatic_data_storage,
+    allow_encrypted_data_storage: this.props.viewer.allow_encrypted_data_storage,
+    changingPassword: false,
+    changingUsername: false,
+    changingAvatar: false,
+    changingFilecoin: false,
+    copyValue: "",
+    loading: false,
+    tab: 0,
+    contextMenu: null,
+    selectedIndex: -1,
+  };
+
+  componentDidMount = () => {
```

Comment:
If you're going to add any event listeners, make sure when the component unmounts you do

```
componentWillUnmount
```

and

```
window.removeEventListener("keydown", this._handleDocumentKeydown)
```